libr/io/p/io_gprobe.c

```diff
@@ -776,6 +891,39 @@ static int gprobe_getdeviceid (struct sp_port *port, ut8 index) {
 	return -1;
 }
 
+static int gprobe_getinformation (struct gport *port) {
+	RBuffer *request = r_buf_new ();
+	RBuffer *reply = r_buf_new ();
+	const ut8 cmd = GPROBE_GET_INFORMATION;
+	const ut8 index = 0;
+	RPrint *print = r_print_new ();
```

Comment:
uhmf. you probably dont want to do that here. r_print_hexdump() works with RPrint instance as NULL